packages/documentation/copy/en/handbook-v2/Type Manipulation/Mapped Types.md

```diff
@@ -5,50 +5,84 @@ permalink: /docs/handbook/2/mapped-types.html
 oneline: "Generating types by re-using an existing type."
 ---
 
-When you don't want to repeat yourself, sometimes a type needs to be based on another type.
+When you don't want to repeat yourself, sometimes a type needs to be based on another type: that is, a new type with the same JavaScript object properties as another type, but modified in some way.
```

Comment:
```suggestion
When you don't want to repeat yourself, sometimes a type needs to be based on another type.
```

I don't think this extra bit adds; in fact, you can make a new object that has entirely different properties which do depend on another type and so it's not "the same object properties" in that way. I think the rest of the section is pretty self explanatory.